CHANGELOG.md

```diff
@@ -7,105 +7,31 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
-- Use correct casing for "learn more" ([#14412](https://github.com/MetaMask/metamask-extension/pull/14412))
-- E2e infura requests ([#14394](https://github.com/MetaMask/metamask-extension/pull/14394))
-- Dark Mode : Fix connected account permission icon color ([#14406](https://github.com/MetaMask/metamask-extension/pull/14406))
-- Update `crowdin/github-action` ([#14381](https://github.com/MetaMask/metamask-extension/pull/14381))
-- Updating info ActionableMessage type to default ([#14385](https://github.com/MetaMask/metamask-extension/pull/14385))
-- Update ganache ([#14375](https://github.com/MetaMask/metamask-extension/pull/14375))
-- Storybook housekeeping 🧹  ([#14331](https://github.com/MetaMask/metamask-extension/pull/14331))
-- Settings search refactor ([#14350](https://github.com/MetaMask/metamask-extension/pull/14350))
-- Update token detection logic to only control auto-detection ([#14251](https://github.com/MetaMask/metamask-extension/pull/14251))
-- Remove `base_url` from crowdin.yml ([#14364](https://github.com/MetaMask/metamask-extension/pull/14364))
-- Master sync following v10.12.4 ([#14373](https://github.com/MetaMask/metamask-extension/pull/14373))
-- TabBar: prevent text from wrapping early ([#14368](https://github.com/MetaMask/metamask-extension/pull/14368))
-- Merge remote-tracking branch 'origin/develop' into master-sync
-- Feature Flag Swap API mock ([#14353](https://github.com/MetaMask/metamask-extension/pull/14353))
-- Settings:TabBar: Fix text wrap style ([#14348](https://github.com/MetaMask/metamask-extension/pull/14348))
-- Dark Mode: What's New Announcement ([#14346](https://github.com/MetaMask/metamask-extension/pull/14346))
-- Dark Mode: Fix Actionable Messages ([#14361](https://github.com/MetaMask/metamask-extension/pull/14361))
-- Fix imported component name and change size proptype type to string ([#14237](https://github.com/MetaMask/metamask-extension/pull/14237))
-- stx-controller-v1.10.0 ([#14352](https://github.com/MetaMask/metamask-extension/pull/14352))
-- Fix the close button in the speed up and cancel popovers ([#14356](https://github.com/MetaMask/metamask-extension/pull/14356))
-- track rpc method usage ([#14269](https://github.com/MetaMask/metamask-extension/pull/14269))
-- Restore `version` missing from certain build steps ([#14344](https://github.com/MetaMask/metamask-extension/pull/14344))
-- Add snap version to details page ([#14110](https://github.com/MetaMask/metamask-extension/pull/14110))
+### Added
+- Dark Mode: Add What's New Announcement ([#14346](https://github.com/MetaMask/metamask-extension/pull/14346))
+- **[FLASK]** Add snap version to details page ([#14110](https://github.com/MetaMask/metamask-extension/pull/14110))
+- **[FLASK]** Add support for searching installed snaps in Settings ([#14419](https://github.com/MetaMask/metamask-extension/pull/14419))
+
+### Changed
+- Updating SRP input on Import SRP page ([#14425](https://github.com/MetaMask/metamask-extension/pull/14425))
```

Comment:
> Updating SRP input on Import SRP page ([#14425](https://github.com/MetaMask/metamask-extension/pull/14425))

Should not be included, it is behind a feature flag

Reply:
Good catch, I've removed this